cpg_workflows/large_cohort/scripts/ancestry_add_background.py

```diff
@@ -122,7 +122,9 @@ def add_background(dense_mt: hl.MatrixTable, sample_qc_ht: hl.Table, tmp_path: s
         # save metadata info before merging dense and background datasets
         sample_qc_ht = sample_qc_ht.union(background_mt.cols(), unify=allow_missing_columns)
 
-        background_mt = background_mt.select_entries('GT', 'GQ', 'DP', 'AD').naive_coalesce(5000)
+        background_mt = (
+            background_mt.select_cols().select_rows().select_entries('GT', 'GQ', 'DP', 'AD').naive_coalesce(5000)
+        )
```

Comment:
As far as I can work out, what this is doing is:
- select_rows with no argument drops any row fields which aren't part of a key - here that would... drop everything except the locus/alleles?
- select_cols with no argument drops any col fields which aren't part of a key, so only keeps sample ID
- selects a portion of the entry fields
- union_cols reduces dense_mt to the sample IDs in the background MT

Maybe? That would mean that the select_rows() call is probably useless, and if that is what's going on here this method should probably be documented better 😅